apps/language_server/lib/language_server/dialyzer/analyzer.ex

```diff
@@ -57,6 +57,10 @@ defmodule ElixirLS.LanguageServer.Dialyzer.Analyzer do
 
     parent = self()
 
+    # dialyzer_analysis_callgraph.start exits on failure
+    # need to unset :trap_exit when analysis finises
+    Process.flag(:trap_exit, true)
+
     pid =
       spawn_link(fn ->
```

Comment:
The alternative to the `trap_exit` would be to use `spawn_monitor` here and receive on the monitor message.